package.json

```diff
@@ -40,7 +40,7 @@
   "prettier": "@bigcommerce/eslint-config/prettier",
   "homepage": "https://github.com/bigcommerce/checkout-js#readme",
   "dependencies": {
-    "@bigcommerce/checkout-sdk": "^1.363.0",
+    "@bigcommerce/checkout-sdk": "^1.366.0",
```

Comment:
There have been quite a few bumps here, for investigating why the e2e test is failing maybe we need to try different bumps and check which PR breaks the test.

Reply:
Hi @animesh1987 
As was found before, e2e tests started failing from build 365 (364.2 was OK)
<img width="1307" alt="Screenshot 2023-03-21 at 10 00 53" src="https://user-images.githubusercontent.com/9430298/226547569-2862b9f1-8b2f-4502-ba7a-3f72861f047f.png">

Reply:
Good debugging, doubt the npm bump caused any issues. Feel might be an issue related to bluesnap. @mauricio-sg can you please look at the test in your related checkout-js Pr and see if we need to update the test.